components/google-cloud/README.md

```diff
@@ -78,4 +78,76 @@ start, let's walk through a simple workflow using this API.
 
     #endpoint
     model_deploy_step.outputs['endpoint_metadata']
-    ```
\ No newline at end of file
+    ```
+
+
+    ### Supported Components 
+    Following is the list of currently supproted components. 
+
+    ```python
+    AutoMLImageTrainingJobRunOp(...)
+        """Runs the AutoML Image training job and returns a model."""
+
+    AutoMLTabularTrainingJobRunOp(...)
+        """Runs the training job and returns a model."""
+
+    AutoMLTextTrainingJobRunOp(...)
+        """Runs the training job and returns a model."""
+
+    AutoMLVideoTrainingJobRunOp(...)
+        """Runs the AutoML Image training job and returns a model."""
+
+    CustomContainerTrainingJobRunOp(...)
+        """Runs the custom training job."""
+
+    CustomPythonPackageTrainingJobRunOp(...)
+        """Runs the custom training job."""
+
+    EndpointCreateOp(...)
+        """Creates a new endpoint."""
+
+    ImageDatasetCreateOp(...)
+        """Creates a new image dataset and optionally imports data into dataset when"""
+
+    ImageDatasetExportDataOp(...)
+        """Exports data to output dir to GCS."""
+
+    ImageDatasetImportDataOp(...)
+        """Upload data to existing managed dataset."""
+
+    ModelBatchPredictOp(...)
+        """Creates a batch prediction job using this Model and outputs prediction"""
+
+    ModelDeployOp(...)
+        """Deploys model to endpoint. Endpoint will be created if unspecified."""
+
+    ModelUploadOp(...)
+        """Uploads a model and returns a Model representing the uploaded Model resource."""
+
+    TabularDatasetCreateOp(...)
+        """Creates a new tabular dataset."""
+
+    TabularDatasetExportDataOp(...)
+        """Exports data to output dir to GCS."""
+
+    TabularDatasetImportDataOp(...)
```

Comment:
This should be removed. Tabular dataset do not support import. I'll remove the op in a follow up PR.

Reply:
got it, I removed the note from read me and the op from Init.